pkg/oauth/interactive.go

```diff
@@ -17,12 +17,59 @@ package oauth
+					</svg>
+				</a>
+			</div>
+			<div>
+				<div class="title">
+					<span class="sigstore">sigstore </span>
+					<span>authentication successful!</span>
+				</div>
+				<div class="content">
+					<span>You may now close this page.</span>
+				</div>
+			</div>
+			<div class="anchor">
+				<div class="links">
+					<a href="https://docs.sigstore.dev/" class="link login"><span class="sigstore">sigstore</span> documentation <span class="arrow">→</span></a>
+					<a href="https://blog.sigstore.dev/" class="link"><span class="sigstore">sigstore</span> blog <span class="arrow">→</span></a>
+					<a href="https://docs.sigstore.dev/gitsign/overview/" class="link login"><span class="sigstore">gitsign</span> documentation <span class="arrow">→</span></a>
```

Comment:
Can we drop this one, since this is documentation specific to just one client within Sigstore?

Reply:
Removed third link and added sigstore home